app/src/main/java/com/github/sdpteam15/polyevents/view/fragments/home/StaffHomeFragment.kt

```diff
@@ -1,13 +1,19 @@
 package com.github.sdpteam15.polyevents.view.fragments.home
 
+import android.content.Intent
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Button
 import androidx.fragment.app.Fragment
 import com.github.sdpteam15.polyevents.R
+import com.github.sdpteam15.polyevents.model.database.remote.Database
 import com.github.sdpteam15.polyevents.model.entity.UserRole
 import com.github.sdpteam15.polyevents.view.activity.MainActivity
+import com.github.sdpteam15.polyevents.view.activity.activityprovider.MyItemRequestsActivity
+import com.github.sdpteam15.polyevents.view.activity.staff.EXTRA_ID_USER_STAFF
+import com.github.sdpteam15.polyevents.view.activity.staff.StaffRequestsActivity
 
 class StaffHomeFragment : Fragment() {
```

Comment:
Could you add a comment here? We probably forgot to write one before

Reply:
True, I chenged it  and there will probably be more comments to add on activities -> last sprint